### ISSUE TITLE
 Unable to install modules islandora, search_api_solr due to missing modules: search_api_solr.

### ISSUE THREAD

**Issue by rosiel** [Islandora-Devops/isle-dc#144]
During `make local`, isle-dc fails because of lacking search_api_solr. 

@ajstanley finds the only way to do this is to watch the filesystem like a hawk during make local, and once the codebase folder is created, go into it and enter `composer require 'drupal/search_api_solr:^1.10'` _while make is busy running_. 

I have not figured out if this is possible yet because I dont have his PHP monitoring script set up. However, cd'ing into codebase, running the composer require, and then attempting `make local` again fails at the same spot.

**Comment by misilot** [comment]
I believe this was fixed in #146 by removing the line `docker-compose exec -T drupal with-contenv bash -lc 'drush -y en islandora search_api_solr'`

**Comment by joecorall** [comment]
The `make local` command has been replaced by documentation.

```
$ make local
make demo has been removed. To create a demo site, please follow the instructions at https://islandora.github.io/documentation/installation/docker-local/
```
